include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -275,6 +275,9 @@
 #    if !(_ONEDPL_FPGA_DEVICE)
 #        define _USE_SUB_GROUPS 1
 #        define _USE_GROUP_ALGOS 1
+#        ifndef _ONEDPL_ICPX_USE_KNOWN_IDENTITY_FOR_ARITHMETIC_64BIT_DATA_TYPES
```

Comment:
For my education, what is the motivation for why this is an important feature to provide for the user?

Do these group reduction operations provide bad performance for 64 bit arithmetic types?

Reply:
They simple provides incorrect results for 64-bit data types.

Reply:
Would it be better then to make the default `0` to avoid using use known identity then, and update the default with a compiler version minimum when it is fixed?

Reply:
I think if we enable something optional by default, we able to write macros like here.
If we not disable something optional by default, we haven't reasons ang proper ways to show in the code that we have this ability. Also I have been implemented proposed by you variant early and it's look not very good when we write all `__has_known_identity` staff. But in any case, it's not the big deal to rewrite this macro by another (disabling) way.

Reply:
Can we just turn off `__has_known_identity` for these types until it is fixed (and update with working compiler version in a future oneDPL release when it is fixed), rather than providing this workaround that requires the user to set this PP definition?

Reply:
Technically we able to do this.
But I believe it's not the best solution: we will not be able to control in tests which functions for which types give errors.
As I know at this moment, for example we able to use `double` with `std::plus`, for example.

Reply:
Ah, I did not realize it was fine-grained and not affecting all 64bit type / operator combinations.  

I still believe the better option is to have the default be correct since we have a codepath which provides correct results, and give an option to "turn on" group reduction operations for those who are interested.

Reply:
After our discussion in the meeting, I'll withdraw this objection to the strategy.  I was thinking this was a compiler issues rather than a driver issue, and there is not a good way to detect this a compile time.  It makes sense to be optimistic in this case and provide a documented workaround in this case.